src/config.rs

```diff
@@ -11,6 +32,15 @@ pub struct Config {
     #[serde(default = "default_device_mode")]
     pub mode: String,
 
+    #[serde(default = "default_unicode")]
+    pub unicode: bool,
+
+    #[serde(default = "default_color_mode")]
+    pub color_mode: ColorMode,
+
+    #[serde(default = "default_narrow_cols")]
+    pub narrow_cols: u16,
```

Comment:
can we rename this so something like `small_layout_breakpoints` that would be a struct with `width` and `height` ? what do you think ?

Reply:
I've added two options, `small_layout_cols` and `small_layout_rows`. I think two separate options are better from the user's perspective, so they don't have to worry about which part of the struct is which.

Reply:
The reason I suggest `height` and `width` is to be consistent with `ratatui` naming already.